flowback/ledger/services.py

```diff
@@ -0,0 +1,94 @@
+from datetime import datetime
+
+from flowback.common.services import model_update, get_object
+from flowback.ledger.models import Account, Transaction
+from flowback.user.models import User
+from django.core.exceptions import ValidationError
+
+
+def account_create(*, account_number: str, account_name: str, user_id: int) -> Account:
+    user = get_object(User, id=user_id)
+    account = Account(account_number=account_number,
+                      account_name=account_name, user=user)
+
+    account.full_clean()
+    account.save()
+
+    return account
+
+
+def account_update(user_id: int, account_id: int, data) -> Account:
+    account = get_object(Account, id=account_id)
+
+    if account.user_id != user_id:
+        raise ValidationError("Account doesn't belong to User")
+
+    data['updated_at'] = datetime.now()
+    non_side_effect_fields = ['account_number', 'account_name', 'updated_at']
+    account, has_updated = model_update(instance=account,
+                                        fields=non_side_effect_fields,
+                                        data=data)
+    return account
+
+
+def account_delete(user_id: int, account_id: int):
+    account = get_object(Account, id=account_id)
+
+    if account.user_id != user_id:
+        raise ValidationError("Account doesn't belong to User")
+
+    account.delete()
+
+
+def transaction_create(*, user_id: int, debit_amount: float = 0, credit_amount: float = 0, description: str, verification_number: str, account_id: int, date: str = datetime.now()) -> Transaction:
```

Comment:
keep lines not too long, you can add enter to each field if nessesary, better to scroll a little more than to scroll horizontally